JASP-Engine/JASP/R/regressionlogistic.R

```diff
@@ -616,27 +616,30 @@ RegressionLogistic <- function(jaspResults, dataset = NULL, options, ...) {
         rn <- rownames(summary(glmObj[[i]])[["coefficients"]])
         rn[which(rn == "(Intercept)")] <- .v("(Intercept)")
         bootname <- paste(rn, collapse = "-")
-
+        
         if (is.null(bootstrapResults[[bootname]])) {
 
           # we compute additional statistics while bootstrapping, but we can't do this using boot
+          # (this is not true, you can boot whatever statistics you want, but ok... the bootstrapping should get some review anyway at some point)
```

Comment:
Just to clarify my comment, the issue was that `boot` isn't really suited to bootstrap both the standardized and unstandardized coefficients at the same time. Mainly because the standardized results can fail without the normal results failing. Rather than bootstrapping twice, I resorted to this approach with an environment. 

I get and agree your comment, but maybe it gets a little bit confusing now for people that read it in the future. Perhaps:
```r
# vandenman: we compute additional statistics while bootstrapping, but we can't do this using boot so we hack it in there using an environment
# Kucharssim (this is not true, you can boot whatever statistics you want, but ok... the bootstrapping should get some review anyway at some point)
```
to clarify this is not a crazy person talking to themself?

Reply:
Yeah, thanks! I am going with the following to make it more clear

```
# vandenman: we compute additional statistics while bootstrapping, but we can't do this using boot
# so we hack it in there using an environment
# kucharssim: this is not true, you can boot whatever statistics you want, but ok... the bootstrapping should get some review anyway at some point
# discussion here: https://github.com/jasp-stats/jasp-desktop/pull/3962#discussion_r394348052
```